### ISSUE TITLE
Create MR with reviewers

### ISSUE THREAD

**Issue by mary3000** [python-gitlab/python-gitlab#1267]
I didn't find an option to assign reviewers while creating MR. 
As I see, gitlab REST API also [doesn't have](https://docs.gitlab.com/ee/api/merge_requests.html#create-mr) this option yet (should I also file an issue about it [here](https://gitlab.com/gitlab-org/gitlab/-/issues)?)

**Comment by JoelFeiner** [comment]
@mary3000 I see `reviewer_ids` in the API. Perhaps the documentation just hadn't been updated yet?

**Comment by m00lecule** [comment]
Hello! I am also struggling to find this feature in `python-gitlab 2.6.0`

**Comment by mattiasb** [comment]
This should be in now since a couple of days. See [here](https://docs.gitlab.com/ee/api/merge_requests.html#update-mr)

**Comment by spyoungtech** [comment]
It would also be nice to be able to change the reviewers on an existing merge request.

**Comment by spyoungtech** [comment]
As a workaround for this, you can do the following:

```python
# update the reviewers on the MR
updated_data = mr.manager.update(mr.get_id(), {'reviewer_ids': [1,2,3]})

#  for completeness, update the object from the server response
if updated_data:
    mr._update_attrs(updated_data)
```

**Comment by mykolasolodukha** [comment]
This might've been closed now since the #1396 is merged.

**Comment by nejch** [comment]
You're right @mykolasolodukha, this does already work. Thanks for the catch!

```python
mr = project.mergerequests.create(
    {
        "source_branch": "your-branch",
        "target_branch": "main",
        "title": "your MR",
        "reviewer_ids": [reviewer_id]
    }
)

mr.reviewer_ids = [reviewer_id, another_reviewer_id]
mr.save()
```
